src/index.html

```diff
@@ -1,12 +1,46 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page__content">
+    <main>
+      <div class="calendar calendar--month-length--29">
+        <div class="calendar__day calendar__day--start-day--sun"></div>
```

Comment:
move start day to container as u did with moth length and use firtdt-child

Reply:
done